[16.0][MIG] purchase_order_secondary_unit: Migration to 16.0

anddago78 opened the pull request:
This module extends the functionality of purchase orders to allow buy products
in secondary unit of distinct category.

MT-3483 @moduon @rafaelbn  @Shide @fcvalgar @yajo please reviews :)

sergio-teruel committed:
purchase_order_secondary_unit: New module to allow buy in secondary units

Nikul-Chaudhary committed:
[MIG] purchase_order_secondary_unit v11 to v12

ps-tubtim committed:
[IMP] purchase_order_secondary_unit: black, isort

ps-tubtim committed:
[MIG] purchase_order_secondary_unit: Migration to 13.0

sergio-teruel committed:
[IMP] purchase_order_secondary_unit: more clear string field
TT27363

sergio-teruel committed:
[IMP] purchase_order_secondary_unit: Less code. Use product_secondary_unit mixin model from product-attribute repository.

kittiu committed:
[13.0][FIX] purchase_order_secondary_unit, error selecting 2nd_uom when no product_id

SurachetS committed:
[MIG] purchase_order_secondary_unit: Migration to 14.0

kakurai8 committed:
Added translation using Weblate (Japanese)

kh-badep committed:
Added translation using Weblate (French)

HviorForgeFlow committed:
[FIX] po_secondary_unit: Drop 0 qty test check on pol with secondary UoM

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (9 of 9 strings)

Translation: purchase-workflow-14.0/purchase-workflow-14.0-purchase_order_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-14-0/purchase-workflow-14-0-purchase_order_secondary_unit/de/

BT-mgomez committed:
[IMP] purchase_order_secondary_unit: black, isort, prettier

BT-mgomez committed:
[15.0][MIG] purchase_order_secondary_unit: Migration to 15.0

Commit:
[IMP] purchase_order_secondary_unit: Compatibility with product variants

Commit:
[UPD] Update purchase_order_secondary_unit.pot

OCA-git-bot committed:
[UPD] README.rst

HviorForgeFlow committed:
[FIX] purchase_order_secondary_unit: Split tests

OCA-git-bot committed:
purchase_order_secondary_unit 15.0.1.0.1

sergio-teruel committed:
[IMP] purchase_order_secondary_unit: Add secondary units to purchase order reports

Commit:
[UPD] Update purchase_order_secondary_unit.pot

OCA-git-bot committed:
purchase_order_secondary_unit 15.0.1.1.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-15.0/purchase-workflow-15.0-purchase_order_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-15-0/purchase-workflow-15-0-purchase_order_secondary_unit/

fcvalgar reviewed (approved):
Functional Review.
LGTM thak you @anddago78

yajo committed:
[MIG] purchase_order_secondary_unit: migrate to v16

Fix a bug when calling super's `_compute_product_qty()` was skipping the compute helper from the mixin.

Fix buggy tests. These were asserting wrong behaviors.

Upgrade views.

@moduon MT-3483

Co-authored-by: Andrea Cattalani <22261939+anddago78@users.noreply.github.com>

sergio-teruel commented:
@anddago78  Why have you closed this PR?
